locales/de-DE.json

```diff
@@ -7,9 +7,9 @@
     "route_loaded": "Seite {0} geladen"
   },
   "account": {
-    "avatar_description": "{0}'s Avatar",
+    "avatar_description": "{0}s Avatar",
```

Comment:
This is not correct, is it?

Das `'` impliziert das es sich um den Besitzer des Avatars handelt

Reply:
"Der Apostroph wird gelegentlich zur Verdeutlichung der Grundform eines Personennamens gebraucht", dass ist hier aber nicht der Fall, daher würde ich darauf verzichten.

Reply:
Oh das kenn ich aber anders
Es ist nicht `Peters Auto`, sondern `Peter's Auto`

Reply:
I learned it where 's is used if it's about someone's something... and a special case with just ' if their name ends with s

Reply:
Die richtige deutsche Rechtschreibung kann man [hier](https://www.duden.de/sprachwissen/sprachratgeber/Ehrenrettung-fur-Grossmutters-Apfelkuchen) nochmal nachlesen. Man macht also niemals ein Apostroph hinter einen Namen, außer dieser endet mit s. Zum Beispiel sind korrekte Schreibweisen: Peters Auto (ohne) oder Jonas' Auto (mit). Alles andere Varianten sind falsch.

Reply:
Und wie kann man conditional i18n anwenden?